addon/components/md-nav.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+import layout from '../templates/components/md-nav';
+
+const { run: { next }, Component } = Ember;
+
+export default Component.extend({
+  classNames: ['md-nav'],
+  layout,
+  tagName: 'nav',
```

Comment:
Wrapping it does work.  I was encountering a bug with liquid-fire where position:fixed does not mesh well with .liquid-container transformY properties.

I have worked around the issue by placing my nav outside the outermost liquid-outlet.

On the original note, I wonder if the component should be a wrapper that can accept `fixed=true`, and the nav be a child element?

> On Jul 19, 2016, at 9:14 PM, Mike North <notifications@github.com> wrote:
> 
> In addon/components/md-nav.js <https://github.com/mike-north/ember-cli-materialize/pull/292#discussion_r71334853>:
> 
> > @@ -0,0 +1,19 @@
> > +import Ember from 'ember';
> > +import layout from '../templates/components/md-nav';
> > +
> > +const { run: { next }, Component } = Ember;
> > +
> > +export default Component.extend({
> > +  classNames: ['md-nav'],
> > +  layout,
> > +  tagName: 'nav',
> navbar-fixed is supposed to wrap the component. It's not a class that's added to the component's root element.
> 
> I would expect this to work
> 
> <div class="navbar-fixed">
>   {{#md-nav}}
>   {{/md-nav}}
> </div>
> If it does not, please create a small example that I can take a look at. A simpler way to debug this kind of thing is to identify where the HTML that this library is building is different than what's specified in the Materialize conventions <http://materializecss.com/navbar.html>.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/mike-north/ember-cli-materialize/pull/292/files/358400ac20e1090d8a515650662d0c7ae61cd446#r71334853>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABCWvbI-wzNIopy5vo7IYmRHbz_5mdagks5qXM3QgaJpZM4HvhE3>.
> 

-- 
Dustin Farris